tmt/steps/provision/__init__.py

```diff
@@ -1306,23 +1333,7 @@ def _check_rsync(self) -> CheckRsyncOutcome:
         except tmt.utils.RunError:
             pass
 
-        # Install under '/root/pkg' for read-only distros
-        # (for now the check is based on 'rpm-ostree' presence)
-        # FIXME: Find a better way how to detect read-only distros
-        # self.debug("Check for a read-only distro.")
-        if self.facts.package_manager == 'rpm-ostree':
-            self.package_manager.install(
-                Package('rsync'),
-                options=tmt.package_managers.Options(
-                    install_root=Path('/root/pkg'),
```

Comment:
Is this really not needed? It seems there is a `--sysroot=SYSROOT` option supported by `rpm-ostree`. What should happen for ostree images where `rsync` is not installed? There definitely was a use case for that before.

Reply:
@psss no it is not needed:

1. first of all, this code never got called in tests, because in all environments `rsync` was available (even RHIVOS), if it would be called, it started to be failing and we would know sooner this part of the code was total bogus
2. special handling is not needed at all, `rsync` gets installed as on normal distros, see [this part of the code](https://github.com/teemtee/tmt/blob/d2b99968e0c07af67ae888ec2c45c9528c531933/tmt/steps/provision/__init__.py#L1336) where it is installed if not available
3. while `sysroot` is available, it is not needed to use it, this is a leftover from the time we were porting from `yum` to `rpm-ostree`, where the leftover options come from

Reply:
Ah, I see. Thanks for the clarification.